package.json

```diff
@@ -221,23 +225,25 @@
     "sinon": "^7.3.1",
     "sinon-chai": "^3.3.0",
     "stream-to-string": "^1.1.0",
-    "tmp": "^0.1.0"
+    "tmp": "^0.1.0",
+    "ts-node": "^8.5.4",
+    "typescript": "^3.7.3"
   },
   "scripts": {
     "build-browser-example": "BABEL_ENV=browser browserify example/index.js -o dist/browser-example.js -t babelify",
     "build-local-continuous": "babel src --watch -d lib --ignore '**/*_spec.js'",
-    "build-local": "babel src -d lib --ignore '**/*_spec.js'",
+    "build-local": "rm -r lib && tsc -p tsconfig.node.json && mv lib/src/* lib/ && rm -r lib/src lib/package.json",
```

Comment:
Could use [del-cli](https://github.com/sindresorhus/del-cli) rather than `rm -r` to keep it cross-platform.

Also could we avoid having to `mv` stuff by tweaking the tsconfig?

Reply:
> Could use del-cli rather than rm -r to keep it cross-platform.

Yep, will probably need that to get appveyor passing.

> Also could we avoid having to mv stuff by tweaking the tsconfig?

Possibly. Wanted lib to have the exact same structure of src and couldn't find a way to do that because we have one file that requires the package.json file

Reply:
Ah, I see. FWIW I had a play locally and was able to remove everything from the `&& mv` onwards by changing the import in `argv_parser.ts` to:

```typescript
const pkg = require('../../package.json') // eslint-disable-line @typescript-eslint/no-var-requires
```

Which...a bit gross, but would avoid the file system stuff at least.

Reply:
I followed this suggestion and think its worth the weirdness. Left a comment as to why we are doing it